webapp/src/components/surveyDateSelector/surveyDateSelector.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useCallback} from 'react';
+import {format} from 'date-fns';
```

Comment:
I wonder if we can/should use intl to format the date. 0/5

Reply:
Done.